.github/workflows/publish-image.yml

```diff
@@ -10,9 +10,15 @@ on:
       additional-latest-tag-name:
         type: string
         default: ''
-      unstable-tag-name:
```

Comment:
This will break existing repos. We should have PRs ready to fix them before merging this